src/root-package-json.js

```diff
@@ -56,9 +56,9 @@ let scripts = {
       'lint:fix': 'npm run lint:fix --workspaces --if-present',
       prepare: 'npm run build',
       start: "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
-      'start:addon': `npm start --workspace ${addonName} -- --no-watch.clearScreen`,
```

Comment:
**question**

What does `--no-watch.clearScreen` do and does it need to be included?

If not really needed, my suggestion is that we remove it to lower maintenance cost (rely little on an external implementation) and match better [what Ember CLI blueprint does](https://github.com/ember-cli/ember-cli/blob/v4.9.2/blueprints/app/files/package.json#L14-L24) (it doesn't add many arguments).

Reply:
it is needed, because when you run multiple watch commands at once, by default, they clear the screen when they re-run, and we don't want that.

[what ember-cli does](https://github.com/ember-cli/ember-cli/blob/v4.9.2/blueprints/app/files/package.json#L14-L24) is insufficient :upside_down_face:

Reply:
Ah, ok. I think it may be better if `@embroider/addon-blueprint` isn't concerned with _how_ an end-user wants to create an addon in a workspace. Nonetheless, I think that my change would be out of scope for this ticket and will add back `--no-watch.clearScreen`.